data/adapters/file_fileset_adapter.py

```diff
@@ -454,20 +468,19 @@ def query_fileset_files_props_encode(self, accession):
         preferred_assay_titles = set()
         assay_term_ids = set()
         method = None
-        class_type = None
+        class_type = 'observed data'
         disease_ids = []
         if dataset_type == 'Annotation':
-            class_type, method, disease_ids = self.parse_annotation(
-                dataset_object, self.portal_url, class_type, method, software, preferred_assay_titles, assay_term_ids, disease_ids)
-            if software and method != 'candidate Cis-Regulatory Elements':
-                software_titles = ', '.join(
-                    sorted([software for software in software]))
-                method = f'{method} using {software_titles}'
-        else:
-            class_type = 'experiment'
+            class_type, method, disease_ids, preferred_assay_titles, assay_term_ids = self.parse_annotation_encode(
+                dataset_object, software)
+            if software and method not in ['candidate Cis-Regulatory Elements', 'caQTL', 'MPRA']:
```

Comment:
is this block just for special case on ENCDOE-rE2G? feel it can be simplified but it works fine